ml-api-adapter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: ml-api-adapter Helm chart for Kubernetes
 name: ml-api-adapter
 version: 9.2.0
```

Comment:
```suggestion
version: 9.5.0
```

Reply:
Sam would we not do this when we are creating a release?

Reply:
I have updated to 9.5.0